compiler/src/dotty/tools/dotc/transform/Splicing.scala

```diff
@@ -60,15 +62,15 @@ object Splicing:
  *  '{ ...
  *     val x1: T1 = ???
  *     val x2: T2 = ???
- *     {{{ 0 | T3 | x1, x2 |
- *       (x1$: Expr[T1], x2$: Expr[T2]) => // body of this lambda does not contain references to x1 or x2
- *         (q: Quotes) ?=> f('{ g(${x1$}, ${x2$}) })
+ *     {{{ 0 | T3 | U, x1, x2 |
+ *       [U$1] => (U$2: Type[U$1], x1$: Expr[T], x2$: Expr[U$1]) => // body of this lambda does not contain references to U, x1 or x2
+ *         (q: Quotes) ?=> f('{ @SplicedType type U$3 = [[[ 0 | U$2 | | U$1 ]]]; g[U$3](${x1$}, ${x2$}) })
```

Comment:
Is the [[[ ... ]]] syntax used here documented somewhere?

Reply:
Not really. It is the syntax defined in the `RefinedPrinter`. I plan to update this syntax in #17144.